pkg/controllers/config/configsync_test.go

```diff
@@ -58,40 +58,61 @@ trust-device-path = /dev/sdb1
 [Global]
 secret-name = openstack-credentials
 secret-namespace = kube-system`,
-			target: `[BlockStorage]
-ignore-volume-az = no
-
-[Global]
+			target: `[Global]
 use-clouds  = true
 clouds-file = /etc/kubernetes/secret/clouds.yaml
 cloud       = openstack`,
+			expectedTopologyValue: "false",
 		}, {
 			name: "Multi-AZ deployment",
 			source: `
 [BlockStorage]
 trust-device-path = /dev/sdb1`,
-			isMultiAZDeployment: true,
-			target: `[BlockStorage]
-ignore-volume-az = yes
-
-[Global]
+			target: `[Global]
 use-clouds  = true
 clouds-file = /etc/kubernetes/secret/clouds.yaml
 cloud       = openstack`,
+			expectedTopologyValue: "false",
 		}, {
 			name: "User-provided AZ configuration is not overridden",
 			source: `
 [BlockStorage]
-trust-device-path = /dev/sdb1
-ignore-volume-az = yes`,
-			isMultiAZDeployment: false,
```

Comment:
What will happen in a z-stream update when there is a ConfigMap with `isMultiAZDeployment: false` / `isMultiAZDeployment: true`? Customers will expect that nothing changes, yet, these unit tests are heavily edited. I miss unit tests that ensure the current behavior, except for cases where it's really a bug.

Reply:
`isMultiAZDeployment` is not a config map key, it is (or rather was) an argument to `translateConfigMap` that is derived by inspecting the Nova and Cinder AZs. This has no user-facing impact.

Reply:
As for why we are removing the `ignore-volume-az` flag: well, it's useless. I documented this [upstream](https://github.com/kubernetes/cloud-provider-openstack/issues/2185) but tl;dr: it applies to late to have any meaningful impact and the only way I can envision it being useful is if the operator sets `[DEFAULT] allow_availability_zone_fallback` in cinder. Even then, they'd be better off simply disabling the feature flag. I actually plan to remove it from upstream Cinder CSI but simply haven't gotten to this yet.